src/Files/Interacts/BaseLayoutCommandImplementationModel.cs

```diff
@@ -787,6 +787,25 @@ public async void DecompressArchiveToChildFolder()
             }
         }
 
+        public async void InstallInfDriver()
+        {
+            foreach (ListedItem selectedItem in SlimContentPage.SelectedItems)
+            {
+                var connection = await AppServiceConnectionHelper.Instance;
```

Comment:
You could move this up so it isn't called everytime as well as the if statement

Reply:
Ah yes, we should probably update the other similar methods like this as well.